sdk/textanalytics/Azure.AI.TextAnalytics/src/TasksStateTasksDetails.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Azure.Core;
+
+namespace Azure.AI.TextAnalytics
+{
+    /// <summary>
+    /// TasksStateTasksDetails.
+    /// </summary>
+    [CodeGenModel("TasksStateTasksDetails")]
+    public partial class TasksStateTasksDetails
```

Comment:
Is this a useful class?

Reply:
no i believe, will make it internal.

Reply:
@mssfang I want to correct myself on this, I  believe this is a useful class as it contains the details of the AnalyzeOperation request, Will be keeping it as public